src/components/Event/EventMenu.svelte

```diff
@@ -1,20 +1,42 @@
 <script>
-    export let event
+    import { fade } from "svelte/transition"
+    import Accordion from "../shared/Accordion.svelte"
+
+    // export let event
     export let setActiveView
+    export let toggleMenu
+    export let menuOpen
 </script>
 
-<ul>
+<div id="EventMenuToggle" on:click={toggleMenu}
+    class="absolute -right-3 -top-0 p-1 bg-white text-emerald-900 border-emerald-900 border cursor-pointer rounded-full h-8 w-8 flex items-center justify-center">
+    <i class="fa fa-arrow-{menuOpen ? 'left' : 'right fa-beat'}"></i>
+</div>
+
+<ul class="mt-8">
     <li>
-        Viewer
+        <i class="fa fa-tv-retro" title="Viewer"></i>
+        <!-- i hate this, native css would allow just toggling one parent selector -->
```

Comment:
this bothers me more than cluttered classnames. using pure Tailwind and no component oriented styles will sometimes require different (and often worse) code logic